lib/resources/r4/procedure.yaml

```diff
@@ -49,3 +52,204 @@ fields:
       - display: Procedure Personnel Relation Type
         system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/388
         info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes/#code-set-388-procedure-personnel-relation-type
+
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  example: |
+    {
+      "resourceType": "Procedure"
+    }
+  url: https://hl7.org/fhir/procedure-definitions.html#Procedure
+  note: resourceType must be Procedure.
+
+- name: subject
+  required: 'Yes'
+  cardinality: 0..1
+  type: Reference
+  description: Indicates the patient who the procedure record is associated with.
+  example: |
+    {
+      "subject": {
+        "reference": "Patient/631923",
+        "display": "Nilsson, Christine"
+      }
+    }
+  url: https://hl7.org/fhir/procedure-definitions.html#Procedure.subject
+  note: subject must be a reference to a Patient.
+
+- name: status
+  required: 'Yes'
+  cardinality: 1..1
+  type: code
+  description: Codes identifying the lifecycle stage of an event.
+  example: |
+    {
+      "status": "completed"
+    }
+  url: https://hl7.org/fhir/r4/procedure-definitions.html#Procedure.status
+  note: Only status code <code>completed</code> is supported.
+
+- name: code
+  required: 'Yes'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: A code to identify a specific procedure.
+  example: |
+    {
+      "coding": [
+        {
+          "system": "http://snomed.info/sct",
+          "code": "119954001",
+          "display": "Adenoid excision (procedure)",
+          "userSelected": true
+        }
+      ],
+      "text": "Adenoid excision (text)"
+    }
+  url: https://hl7.org/fhir/procedure-definitions.html#Procedure.code
+  note: A code must be provided.
+
+- name: encounter
+  required: 'No'
+  cardinality: 0..1
+  type: Reference
+  description: Indicates the encounter when the procedure was performed.
+  example: |
+    {
+      "encounter": {
+        "reference":"Encounter/26070829"
+      }
+    }
+  url: https://hl7.org/fhir/procedure-definitions.html#Procedure.encounter
```

Comment:
missing R4

Reply:
and below

Reply:
Updated [here](https://github.com/cerner/fhir.cerner.com/pull/372/commits/edd713404a0347648bfd408e254be8a2d11466ce#diff-80d4a0c17acadba0cd26a39c2cfeaf8dR111).